packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx

```diff
@@ -74,112 +67,110 @@ export const EmbedExternalContentModal = ({ onClose }: EmbedExternalContentModal
     }
   }, [mainInputRef]);
 
-  return (
```

Comment:
It seems that you are only removing the `()` here
Why?

Reply:
hmm, you are right. I have run the codemod, didn't remove it manually. Something is wrong with the codemod? hmmmm, gonna check it. thanks!

Reply:
try running the `prettier`, the default of the jscodeshift is not wrap single components in `()` in this case the single element being the `Fragment`.
But we have configured in prettier that it needs to, so prettier should fix it